templates/index.html

```diff
@@ -23,15 +23,14 @@ <h1 >You find yourself in a verdant forest.</h1>
                     <input id="loginFormInput" type="text" placeholder="You struggle to recall your name." />
                     <input type="submit" value="Remember" />
                 </form>
-            </div> 
+            </div>
 
             <!-- Chat could be moved into canvas later, if needed. -->
             <div id="chat" class="text">
                 <div class="messages" id="messages"></div>
 
                     <form autocomplete="off" id="chatForm" class="chat-form">
                         <input type="text" id="chatMessageInput" placeholder="Enter a chat message" />
-                        <input type="submit" value="Send" />
```

Comment:
@bdjewkes - just an idea, since hitting tab and then enter creates another situation to keep track of. :) Let me know what you think.

Reply:
Taking it out seems like a good way to simplify the input concerns. Does the `submit` event still fire, though? I think its still subscribed to from `initializeChatListener`; I'm curious if this works

Reply:
(just tested locally) -- I think it works great, and the css highlighting really helps.  :+1: